packages/types/src/CTypeMetadata.ts

```diff
@@ -34,5 +36,5 @@ export interface IMetadata {
 
 export interface ICTypeMetadata {
   metadata: IMetadata
-  ctypeHash: string | null
+  ctypeHash: HexString | null
```

Comment:
This could probably refer to `ICtype['hash']`, or viceversa.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/505/commits/6f20a1067af5155eb85d510a308bdf96c8d58e21